managed/services/management/backup/backups_service_test.go

```diff
@@ -205,103 +248,155 @@ func TestScheduledBackups(t *testing.T) {
-			CronExpression: "* * * * *",
-			Type:           models.ScheduledMySQLBackupTask,
-		})
-		require.NoError(t, err)
+			artifacts, err := models.FindArtifacts(db.Querier, models.ArtifactFilters{
+				ScheduleID: id,
+			})
 
-		id := task.ID
-
-		_, err = models.CreateArtifact(db.Querier, models.CreateArtifactParams{
-			Name:       "artifact",
-			Vendor:     "mysql",
-			LocationID: locationRes.ID,
-			ServiceID:  *agent.ServiceID,
-			DataModel:  models.PhysicalDataModel,
-			Mode:       models.Snapshot,
-			Status:     models.PendingBackupStatus,
-			ScheduleID: id,
+			assert.NoError(t, err)
+			assert.Len(t, artifacts, 0)
 		})
-		require.NoError(t, err)
+	})
 
-		_, err = backupSvc.RemoveScheduledBackup(ctx, &backupv1beta1.RemoveScheduledBackupRequest{
-			ScheduledBackupId: task.ID,
-		})
-		assert.NoError(t, err)
+	t.Run("mongo", func(t *testing.T) {
+		agent := setup(t, db.Querier, models.MongoDBServiceType, t.Name())
 
-		task, err = models.FindScheduledTaskByID(db.Querier, task.ID)
-		assert.Nil(t, task)
-		tests.AssertGRPCError(t, status.Newf(codes.NotFound, `ScheduledTask with ID "%s" not found.`, id), err)
+		t.Run("scheduling physical backups fail when PITR is enabled", func(t *testing.T) {
+			t.Parallel()
+			ctx := context.Background()
+			schedulerService := &mockScheduleService{}
+			backupSvc := NewBackupsService(db, nil, schedulerService)
 
-		artifacts, err := models.FindArtifacts(db.Querier, models.ArtifactFilters{
-			ScheduleID: id,
+			schedulerService.On("Add", mock.Anything, mock.Anything).Return("", nil)
+			_, err := backupSvc.ScheduleBackup(ctx, &backupv1beta1.ScheduleBackupRequest{
+				ServiceId:     *agent.ServiceID,
+				LocationId:    locationRes.ID,
+				Name:          "name",
+				Description:   "description",
+				RetryInterval: durationpb.New(maxRetryInterval),
+				Retries:       maxRetriesAttempts,
+				DataModel:     backupv1beta1.DataModel_PHYSICAL,
+				Mode:          backupv1beta1.BackupMode_PITR,
+			})
+			require.Error(t, err)
+			assert.Contains(t, err.Error(), backup.ErrIncompatibleDataModel.Error())
 		})
 
-		assert.NoError(t, err)
-		assert.Len(t, artifacts, 0)
+		t.Run("scheduling physical backups snapshot is successful", func(t *testing.T) {
+			t.Parallel()
+			ctx := context.Background()
+			schedulerService := &mockScheduleService{}
+			backupSvc := NewBackupsService(db, nil, schedulerService)
+			schedulerService.On("Add", mock.Anything, mock.Anything).Return(&models.ScheduledTask{}, nil)
+			_, err := backupSvc.ScheduleBackup(ctx, &backupv1beta1.ScheduleBackupRequest{
+				ServiceId:     *agent.ServiceID,
+				LocationId:    locationRes.ID,
+				Name:          "name",
+				Description:   "description",
+				RetryInterval: durationpb.New(maxRetryInterval),
+				Retries:       maxRetriesAttempts,
+				DataModel:     backupv1beta1.DataModel_PHYSICAL,
+				Mode:          backupv1beta1.BackupMode_SNAPSHOT,
+			})
+			require.NoError(t, err)
+		})
```

Comment:
I think we shouldn't run such tests in parallel because they can affect each other.

Reply:
none of the tests are modifying data, they're  reading (and checking error conditions/persisting to db), so I'm not sure they affect each other in this case.